builtin/tests/builtin_actors_tests.rs

```diff
@@ -14,3 +15,39 @@ fn change_owner_success() {
     let w = setup();
     change_owner_success_test(&w);
 }
+
+#[test]
+fn account_authenticate_message() {
+    let w = setup();
+    account_authenticate_message_test(&w);
+}
+
+#[test]
+fn eth_test() {
+    let w = setup();
+    placeholder_deploy_test(&w);
```

Comment:
this needs to patch out to a builtin-actors branch with the test fixed to use execute_implicit for the call from EAM_ACTOR. 

that still doesn't fully fix the test however (see #40)